### PR TITLE
Add a cloudbuild.yaml for tag releases.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by dlorenc** [comment]
This should finish up the release automation.

File: cloudbuild-release.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This cloudbuild is run on the creation of new tags, which should signify releases.
+steps:
+# Build the container that will do the go build.
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build', '-t', 'builder', '-f', 'Dockerfile.build', '.']
```
Comment (nkubala) [inline cloudbuild-release.yaml]: I don't see `Dockerfile.build` anywhere, do you need to check it in?

Comment (dlorenc) [reply cloudbuild-release.yaml]: It should have just been added in https://github.com/GoogleCloudPlatform/container-diff/commit/b726215b8b978e1d85257af5c2fd0a6fb8e116fd

Comment (nkubala) [reply cloudbuild-release.yaml]: Ah yep looks like my pull didn't finish before I said that, spoke too soon :)

**Comment by termux-io** [comment]
`00d283c33d5faade8e88f718a041083230658a49`

**Pull request merged**